ui/tests/helpers/components/dashboard/dashboard-selectors.js

```diff
@@ -0,0 +1,3 @@
+export const SELECTORS = {
+  cardName: (name) => `[data-test-card="${name}"]`,
```

Comment:
I will update my other acceptance / integration tests to use general selectors in a different PR!